main.cpp

```diff
@@ -179,14 +192,31 @@ void run()
   check_solution<T>(num_times, a, b, c, sum);
 
   // Display timing results
-  std::cout
-    << std::left << std::setw(12) << "Function"
-    << std::left << std::setw(12) << "MBytes/sec"
-    << std::left << std::setw(12) << "Min (sec)"
-    << std::left << std::setw(12) << "Max"
-    << std::left << std::setw(12) << "Average" << std::endl;
+  if (output_as_csv)
+  {
+    std::cout
+      << "function"  << csv_separator
+      << "nreps"  << csv_separator
```

Comment:
Can you change this to be `num_times` please? It's more consistent with the CLI arguments and the style of using underscores to separate words in the header.